### ISSUE TITLE
Can't Add Tabs (After The First One)

### ISSUE THREAD

**Issue by yussufbiyik** [sentialx/multrin#118]
**Bug description**
I can't add a second window to Multrin

**To Reproduce**
Dragging windows to Multrin is enough to produce the bug

**Expected behavior**
I was expecting to add my window to Multrin as a second tab

**Screenshots**
![gif](https://dev-to-uploads.s3.amazonaws.com/i/7nksyic0kirdkpf2bwqz.gif)
Gif Link -> https://dev-to-uploads.s3.amazonaws.com/i/7nksyic0kirdkpf2bwqz.gif
**Details:**
 - Operating System: Windows 10 Pro
 - Multrin version: 1.3.0
 - Last known working Multrin version: Just downloaded
 - Scaling (in percentage, if present): 1 screen, 175%
**Additional context**
I also can't dock windows in Multrin and I can produce this bug the way this is produced

**Comment by yoyowu1000** [comment]
Yeah have a similar issue. I can still drag, but it has to be like precise.
